tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -310,3 +310,185 @@ user enters text into meta guidance data file content editor
+    user waits until h2 is visible  Uploaded data files
+    user waits until page contains accordion section   ${SUBJECT_NAME}
+    user opens accordion section   ${SUBJECT_NAME}
+    ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
+    user checks headed table body row contains  Status           Complete  ${section}  180
+
+user waits for release to publish 
+    # EES-1007 - Release process status doesn't automatically update
+    user waits until h2 is visible  Sign off
+    user checks summary list contains  Current status  Approved
+    user waits for release process status to be  Complete    ${release_complete_wait}
+    user reloads page  # EES-1448
+    user checks page does not contain button  Edit release status
+
+user navigates to Data blocks section 
+    user clicks link    Data blocks
+    user waits until h2 is visible  Data blocks
+    user clicks link  Create data block
+    user waits until h2 is visible  Create data block
+    user waits until h2 is visible   Choose a subject
+
+user approves release for scheduled release 
+    [Arguments]  ${RELEASE_MONTH}  ${RELEASE_YEAR}
+    ${PUBLISH_DATE_DAY}=  get current datetime  %-d
+    ${PUBLISH_DATE_MONTH}=  get current datetime  %-m
+    ${PUBLISH_DATE_MONTH_WORD}=  get current datetime  %B
+    ${PUBLISH_DATE_YEAR}=  get current datetime  %Y
+    set suite variable  ${PUBLISH_DATE_DAY}
+    set suite variable  ${PUBLISH_DATE_MONTH}
+    set suite variable  ${PUBLISH_DATE_MONTH_WORD}
+    set suite variable  ${PUBLISH_DATE_YEAR}
+
+    user clicks button  Edit release status
+    user waits until h2 is visible  Edit release status
+
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote  Approved by UI tests
+    user clicks radio  As soon as possible
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-month   ${RELEASE_MONTH}
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-year    ${RELEASE_YEAR}
+
+    user clicks button   Update status
+
+user waits for scheduled release to be complete 
+    [Arguments]  ${PUBLISH_DAY}  ${PUBLISH_MONTH}  ${PUBLISH_YEAR}
+    # EES-1007 - Release process status doesn't automatically update
+    user waits until h2 is visible  Sign off
+    user checks summary list contains  Current status  Approved
+    user checks summary list contains  Scheduled release  ${PUBLISH_DAY} ${PUBLISH_MONTH} ${PUBLISH_YEAR}
+    user waits for release process status to be  Complete    ${release_complete_wait}
+    user reloads page  # EES-1448
+    user checks page does not contain button  Edit release status
+
+user creates new content section
+    [Arguments]  ${PUBLICATION_NAME}  ${CONTENT_SECTION_NAME}
+    user waits until h2 is visible  ${PUBLICATION_NAME}
+    user clicks button  Add new section
+    user changes accordion section title  1   ${CONTENT_SECTION_NAME}
+
+user verifies release summary
```

Comment:
There are multiple test suites that can now use this keyword - you can find them by searching for `user checks summary list contains` and look for ones using `Publication title`